test/e2e/atlas_users_test.go

```diff
@@ -96,6 +96,10 @@ func TestAtlasUsers(t *testing.T) {
 
 	g.Run("Invite", func(t *testing.T) { //nolint:thelper // g.Run replaces t.Run
 		n := g.memoryRand("rand", 10000)
+		if IsGov() {
```

Comment:
move it before generating the random number, no?

Reply:
done